CMakeLists.txt

```diff
@@ -63,3 +63,17 @@ if (pfs_BUILD_TESTS)
     add_executable (unittest ${pfs_UNITTEST_SOURCES})
     target_link_libraries (unittest PRIVATE pfs)
 endif()
+
```

Comment:
General comment. Can you please add a few lines to the `README.md` file to explain how a project is supposed to use `find_package`?
I guess a simple paragraph under the `Integrate` section should suffice. Something like this maybe? (Trying to reduce the load from you)
```
### Use CMake's [find_package()](https://cmake.org/cmake/help/latest/command/find_package.html)

After building the project, you can install it locally using `make install`.
In your project's CMake file, you can add:

    find_package(pfs)
    include_directories (${pfs_INCLUDE_DIRS})
    target_link_libraries (<your-target> pfs)

And CMake should handle the rest.
```